libnavigation-core/src/test/java/com/mapbox/navigation/core/routerefresh/RouteRefreshControllerTest.kt

```diff
@@ -188,20 +218,24 @@ class RouteRefreshControllerTest {
     }
 
     @Test
-    fun `do not send a request when uuid is empty`() = runBlockingTest {
-        val routeRefresh = mockk<RouteRefresh>(relaxed = true)
-        val routeRefreshController = createRouteRefreshController(
-            routeRefresh = routeRefresh
-        )
-        val route = createNavigationRoute(createTestTwoLegRoute(requestUuid = ""))
+    fun `do not send a request when uuid is empty, observer is idle`() =
```

Comment:
```suggestion
    fun `when uuid is empty, controller has an idle state, request't is not sent`() =
```

Reply:
I would put info about observer to the end of sentence as it's does in other places

Reply:
I did it in this way as I value readability more than consistency. But I don't think this question is too important to waste your time in this discussion, feel free to do as you like.